### PR TITLE
Makefile: Add `ubuntu` to supported BLD_ARCH

### PR DESCRIPTION
**Pull request by goutamtadi1**
- BLD_ARCH for ubuntu is ubuntu1604_amd64

### CODE REVIEW

File: gpAux/Makefile [hunk 1]
```diff
@@ -174,6 +174,7 @@ suse11_x86_64_CONFIGFLAGS=--enable-gpperfmon --with-gssapi ${ORCA_CONFIG} --with
 sles11_x86_64_CONFIGFLAGS=--enable-gpperfmon --with-gssapi ${ORCA_CONFIG} --with-libxml $(APR_CONFIG) $(APU_CONFIG)
 linux_x86_64_CONFIGFLAGS=--enable-snmp ${ORCA_CONFIG} --with-libxml $(APR_CONFIG)
 osx106_x86_CONFIGFLAGS=--enable-snmp ${ORCA_CONFIG} --with-libxml $(APR_CONFIG)
+ubuntu1604_amd64_CONFIGFLAGS=--enable-snmp --enable-tap-tests --enable-gpperfmon --with-gssapi --enable-mapreduce ${ORCA_CONFIG} --with-libxml $(APR_CONFIG) $(APU_CONFIG)
```
Comment (goutamtadi1) [inline gpAux/Makefile]: @xinzweb Need some feedback regarding all the flags that are copied here from rhel7 Config flags. We don't know if all of these are required to be applied now.

Comment (xinzweb) [reply gpAux/Makefile]: Probably @d will have more insight on those configure flags. I would do the same, keep all the existing configuration flags unless they are failing.

Actually, a separate movement will try to make all the useful configure flags by default, then we can just keep the list short. However, that's out of scope of this commit.

Comment (larham) [reply gpAux/Makefile]: I vote to omit all flags until PM stories require them.

Comment (jpatel-pivotal) [reply gpAux/Makefile]: I vote to keep list of tags short and let us add to it as/when stories/chores/tests lead us to them.

**Pull request merged**